gen-compdocs/generators/doc.go

```diff
@@ -412,13 +418,33 @@ func UnquoteUsage(flag *pflag.Flag) (name string, usage string) {
 	switch name {
 	case "bool":
 		name = ""
-	case "float64":
+	case "float64", "float32":
 		name = "float"
-	case "int64":
+	case "int64", "severity", "Level":
 		name = "int"
 	case "uint64":
 		name = "uint"
+	case "stringSlice", "stringArray":
+		name = "strings"
+	case "intSlice":
+		name = "ints"
+	case "uintSlice":
+		name = "uints"
+	case "boolSlice":
+		name = "bools"
+	case "mapStringString":
+		name = "&lt;comma-separated 'key=value' pairs&gt;"
+	case "mapStringBool":
+		name = "&lt;comma-separated 'key=True|False' pairs&gt;"
+	case "namedCertKey", "ip":
+		name = "string"
+	case "moduleSpec":
+		name = "&lt;comma-separated 'pattern=N' settings&gt;"
+	case "portRange":
+		name = "&lt;a string in the form 'N1-N2'&gt;"
+	case "traceLocation":
+		name = "&lt;a string in the form 'file:N'&gt;"
 	}
-
+	fmt.Printf("type name = %s\n", name)
```

Comment:
Do you want to keep this line?
https://travis-ci.org/github/kubernetes-sigs/reference-docs/builds/760101045